drivers/max31865/include/max31865_params.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2025 David Picard
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#pragma once
+
+/**
+ * @ingroup     drivers_max31865
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for the MAX31865 driver
+ *
+ * @author      David Picard
+ */
+
+#include "board.h"
+#include "saul_reg.h"
```

Comment:
To make `ARRAY_SIZE` known:
```suggestion
#include "board.h"
#include "container.h"
#include "saul_reg.h"
```

Reply:
Shouldn't `container.h` be included in `max31865_lut_def.h` instead? This is the only place where it's used.

Reply:
:thinking: Good question. The best way would be to generate the `#include` directive with `genlut.py` which also generates `max31865_lut_def.h` so that custom LUTs also have it.